include/Photo.php

```diff
@@ -847,9 +855,30 @@ function import_profile_photo($photo, $uid, $cid, $quit_on_error = false) {
 			$photo_failure = true;
 		}
 
-		$photo = App::get_baseurl() . '/photo/' . $hash . '-4.' . $img->getExt();
-		$thumb = App::get_baseurl() . '/photo/' . $hash . '-5.' . $img->getExt();
-		$micro = App::get_baseurl() . '/photo/' . $hash . '-6.' . $img->getExt();
+		$suffix = '?ts='.time();
```

Comment:
It's a good idea to use the timestamp to refresh cache, can you please include it in the filename instead of a GET parameter?